### DISCUSSION TITLE
Enterprise Support for Concourse?

### DISCUSSION THREAD

**Discussion by richo6030** [concourse/concourse#9034] [Help & Support] [answered]
Hi everyone,

I'm looking for someone who could shed some light on a question relate to support.
We use Concourse extensively in our organization, and we’re looking for potential enterprise-level support options. Specifically, we're interested in knowing if there are any companies out there or services that provide dedicated development support for Concourse.

For example, if an urgent fix or enhancement is required, we would want the provider to be able to quickly assess the issue, provide a PR to the community, and help drive it forward with the community itself based on the guidelines. 🚀

I follow the Concourse discussions and issues here on a daily basis and am aware of the current situation, including the stuff surrounding Broadcom. However, we are still looking for any recommendations or insights from anyone who might know of companies offering this kind of support, just to have it on the table as a possibility. 🙏

Thanks in advance for any help!

**Comment by taylorsilva** [comment, marked as answer]
As far as I know there are no companies providing development support for Concourse _right now_. I am looking to change that though, starting in the new year. Expect news in the new year!

**Comment by richo6030** [comment]
Thank you for the reply back @taylorsilva,

I will wait for the update from your side then. Marking this an answered question, but feel free to add an update in 2025 on the topic.
Happy holidays.
